experimental/responsive-design/src/input.css

```diff
@@ -0,0 +1,29 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@layer base {
+    h1 {
+        @apply text-lg;
+    }
+    :is(h2, h3) {
+        @apply text-base;
+    }
+    :is(h1, h2, h3, h4) {
+        @apply text-gray-700;
+        @apply font-bold;
+    }
+}
+
+@layer utilities {
+    .content-auto {
+        content-visibility: auto;
```

Comment:
Is `content-visibility` widely supported across browsers and used on popular sites?

Reply:
Actually it seems supported partially in Safari.

From the chromestatus page, can we see which different values are used and not just the property?

Reply:
"auto" seems to be supported as of Safari 26 (2025-09-15⁩) so it's pretty new. Do we want to keep it?